packages/itwinui-react/src/core/Table/Table.tsx

```diff
@@ -1060,7 +1094,14 @@ export const Table = <
             </Box>
           )}
           {!isLoading && data.length === 0 && !areFiltersSet && (
-            <Box className='iui-table-empty'>
+            <Box
+              as='div'
+              {...emptyTableContentProps}
+              className={cx(
+                'iui-table-empty',
```

Comment:
there are multiple instances of the iui-table-empty div. all of them should be updated.

Reply:
Done!